extensions/wikia/AssetsManager/config.php

```diff
@@ -215,6 +215,7 @@
 		'//extensions/wikia/AdEngine/js/context/adContext.js',
 		'//extensions/wikia/AdEngine/js/lookup/lookupFactory.js',
 		'//extensions/wikia/AdEngine/js/slot/adSlot.js',
+		'//extensions/wikia/AdEngine/js/slot/adUnitBuilder.js',
```

Comment:
@bognix it's also used by https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js#L6